docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+        required_range:
+          min: 300.0
+          max: 700.0
+        input_processing:
+          - remove_duplicates
+          - sort
+      - name: photo_detection_efficiency
+        description: Average quantum or photon detection efficiency.
+        required: true
+        unit: dimensionless
+        type: double
+        allowed_range:
+          min: 0.0
+          max: 1.0
+```
+
+Input processing in form of sorting, removing of duplicates, etc. can be specified using the `input_processing` field.
+
+:::{Warning}
+The `data_table` field is not yet implemented in the simtools schema and parameter files.
+:::
+
+#### Instrument description
+
+- `class`: Instrument class. Allow values are *Camera*, *Site*, *Structure*, *Telescope*
+- `type`: Instrument type following CTAO Naming.
+- `site`: CTAO site. Allowed values are *North* and *South*.
+
+### Activity description
+
+Describes setting and validation activities. Each activity corresponds to a workflow as described in the [simtools workflows repository](https://github.com/gammasim/workflows).
+
+:::{Warning}
+The implementation of activities and workflows is incomplete and in progress.
+:::
+
+### Data source description
+
+Describes the source of the data or parameter (e.g., *Calibration*)
+
+### Simulation software description
+
+Describes the simulation software (e.g., *sim_telarray* or *corsika*) the parameter is used for.
+
+```yaml
+simulation_software:
+  - name: corsika
+```
+
+Allowed is here to add the internal simulation software parameter naming (if different than the name used here in the schema).
+This is important to write correct and consistent CORSIKA or sim\_telarray configuration files.
+
+ Example:
+
+```yaml
+simulation_software:
+  - name: sim_telarray
+    internal_parameter_name: secondary_ref_radius
+```
+
+## Data structure for model parameters
+
+The model parameters are stored in json-style in the model repository and database.
+A typical model parameter file looks like:
+
+```json
+{
+    "parameter": "num_gains",
+    "instrument": "LSTN-01",
+    "site": "North",
+    "version": "2024-02-01",
+    "value": 2,
+    "unit": null,
+    "type": "int64",
+    "applicable": true,
+    "file": false
+}
+```
+
+## Updating the model database
+
+:::{Danger}
+This is for experts only and might impact the simulation model database for all users.
+Tests should be done before applying this to the production databases.
+:::
+
+### Update a single model parameter
+
+New model parameter defined in the simtools database format (json file) can be uploaded to the database using the {ref}`simtools-add-value-from-json-to-db <add_value_from_json_to_db>` application.
+New data files can be uploaded using {ref}`simtools-add-file-to-db <add_file_to_db>`.
+
+### Upload a complete simulation model from model repository to model database
+
+The application `add_model_parameters_from_repository_to_db.py` allows to export the simulation model parameters from a
```

Comment:
Is there a validation step running in the background whenever we try to upload file or repository to the DB? Either way it is worth mentioning it here.